schemas/meshTriangleCount.schema.json

```diff
@@ -7,113 +7,163 @@
   "properties": {
     "beard": {
       "type": "integer",
+      "minimum": 1,
       "maximum": 1000,
       "errorMessage": {
+        "minimum": "Too few triangles! This asset must have at least 1 triangle.",
         "maximum": "Mesh ${1/name} exceeds triangle count budget. Allowed: 1000. Found: ${0}."
       }
     },
     "body": {
       "type": "integer",
+      "minimum": 1,
       "maximum": 14000,
       "errorMessage": {
+        "minimum": "Too few triangles! This asset must have at least 1 triangle.",
         "maximum": "Mesh ${1/name} exceeds triangle count budget. Allowed: 14000. Found: ${0}."
       }
     },
     "bodyCustom": {
       "type": "integer",
+      "minimum": 1,
       "maximum": 14000,
       "errorMessage": {
+        "minimum": "Too few triangles! This asset must have at least 1 triangle.",
         "maximum": "Mesh ${1/name} exceeds triangle count budget. Allowed: 14000. Found: ${0}."
       }
     },
```

Comment:
bodyCustom is not in use anymore, right? since we don't check it in costumes anymore. Let's remove it from the schema then.